templates/default/config.php.erb

```diff
@@ -47,3 +47,7 @@ $config['rrdtool_version'] = '1.5.5';
 
 $config['geoloc']['latlng'] = true;
 $config['geoloc']['engine'] = "google"; //Only one available at present
+
+<% if @add_conf_enabled == true %>
+ include "<%= @add_conf_file_path %>
```

Comment:
can you check if the file exist ?
Can you add __DIR__ to the config file

`if(file_exists( __DIR__  . '<%= @add_conf_file_path %>')) {`
`    include  __DIR__  . '<%= @add_conf_file_path %>';`
`}`